lib/runner/cli/cli.js

```diff
@@ -326,15 +359,25 @@ class CliRunner {
   }
 
   isTestWorkersEnabled() {
-    return this.test_settings.testWorkersEnabled && !singleSourceFile(this.argv);
+    const testWorkers =  this.test_settings.testWorkersEnabled && !singleSourceFile(this.argv);
+    if (!testWorkers) {
+      return false;
+    } else if (this.isMobile) {
```

Comment:
@gravityvi keep the util functions as they were. Instead writing your own isMobile function, write the for loop here and pass in the `desiredCapabilities` to the `isMobile` function from Utils.